### PR TITLE
Make fallback locale configurable

### PR DESCRIPTION
**Pull request by lucko**

### CODE REVIEW

File: api/src/main/java/net/kyori/adventure/translation/TranslationRegistryImpl.java [hunk 1]
```diff
@@ -36,8 +37,23 @@
 import static java.util.Objects.requireNonNull;
 
 final class TranslationRegistryImpl implements Examinable, TranslationRegistry {
+  private static final Supplier<Locale> SYSTEM_DEFAULT_LOCALE;
+  static {
+    final String property = System.getProperty("net.kyori.adventure.defaultLocale");
```
Comment (kashike) [inline api/src/main/java/net/kyori/adventure/translation/TranslationRegistryImpl.java:42]:
```suggestion
    final String property = System.getProperty("net.kyo".concat("ri.adventure.defaultLocale");
```

**Review by kashike — CHANGES_REQUESTED** [review]

**Comment by Electroid** [comment]
I don't think this will work with a global translator -- if 1 plugin changes the default, it's likely to break another plugin that depends on another default.

**Comment by lucko** [comment]
@Electroid Yep, I agree - I made another PR #167 to resolve this.

**Comment by kashike** [comment]
#179

**Pull request closed without merging**